modules/graph/fragment/gar_fragment_builder_impl.h

```diff
@@ -120,7 +117,8 @@ boost::leaf::result<void> generate_csr_for_reused_edge_label(
       },
       concurrency);
 
-  VLOG(100) << "Finish building the CSR ..." << get_rss_pretty()
+  VLOG(100) << "[frag-" << this->fid_
+            << "] RSS after building the CSR ..." << get_rss_pretty()
```

Comment:
Could you  help to unify the similar logs in `arrow_fragment_builder_impl.h` as well?

Reply:
OK